pages/curriculum.tsx

```diff
@@ -57,17 +55,58 @@ export const Curriculum: React.FC<{}> = () => {
       return !passed
     })
   )
-
   // Progress Percentage should be calculated from lessons 0-6 because thats our current standard of finishing the curriculum.
   const TOTAL_LESSONS = 7
-  const progressPercentage = Math.floor(
-    (lessonInProgressIdx * 100) / TOTAL_LESSONS
+  return Math.floor((lessonInProgressIdx * 100) / TOTAL_LESSONS)
+}
+const calculateCurrent = (session: Session, lessons: Lesson[]): number => {
+  const lessonStatusMap = generateMap(session)
+  return _.cond([
+    [_.isEqual.bind(null, -1), _.constant(0)],
+    [_.constant(true), (output: number) => output]
+  ])(
+    lessons.findIndex(lesson => {
+      const lessonId = _.get(lesson, 'id', '-1') as string
+      const passed = _.get(lessonStatusMap[lessonId], 'isPassed', false)
+      return !passed
+    })
   )
+}
+const DiscordWidget = () => (
+  <iframe
+    className="mt-3 w-100"
+    src="https://discord.com/widget?id=828783458469675019&theme=light"
+    height="500"
+    allowTransparency
+    frameBorder="0"
+    sandbox="allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts"
+  ></iframe>
+)
+export const Curriculum: React.FC<Props> = ({ lessons, alerts }) => {
+  const { data } = useGetSessionQuery({ fetchPolicy: 'cache-and-network' })
```

Comment:
I'm torn on adding this policy. It's not needed in 99% of cases but if user disabled localStorage refreshing the page will reset his status back to unauthorized. 
It won't cause any problems for normal users except small additional traffic.